### PR TITLE
[App-2465] Update UI of the see more screen with the items

### PR DESCRIPTION
**Pull request by ricardo-frango**
**What does this PR do?**

   Updated the "See all" section.
   Also update the installation progress text.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] MoreBundleView.kt
- [ ] ProgressText.kt

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APP-2465](https://aptoide.atlassian.net/browse/APP-2465)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2465](https://aptoide.atlassian.net/browse/APP-2465)

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[APP-2465]: https://aptoide.atlassian.net/browse/APP-2465?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2465]: https://aptoide.atlassian.net/browse/APP-2465?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MoreBundleView.kt [hunk 1]
```diff
@@ -143,26 +142,35 @@ fun AppsList(
...
-          onClick = {},
-          style = Default(fillWidth = false),
-        )
+        InstallViewShort(app)
       }
     }
   }
 }
 
 @PreviewAll
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MoreBundleView.kt:156]: Lets use `PreviewDark`

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MoreBundleView.kt [hunk 2]
```diff
@@ -143,26 +142,35 @@ fun AppsList(
...
+      title = getRandomString(range = 1..5, capitalize = true),
+      reload = {},
+      noNetworkReload = {},
+      navigateBack = {},
+      navigate = {},
+    )
+  }
+}
+
+@PreviewAll
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MoreBundleView.kt:171]: Lets use `PreviewDark`

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MoreBundleView.kt [hunk 3]
```diff
@@ -179,10 +187,10 @@ fun MoreBundleViewPreview(
 
 class AppsListUiStateProvider : PreviewParameterProvider<AppsListUiState> {
   override val values: Sequence<AppsListUiState> = sequenceOf(
-    AppsListUiState.Idle(List(size = Random.nextInt(15)) { randomApp }),
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MoreBundleView.kt:182]: Can we avoid this change?

File: app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt [hunk 4]
```diff
@@ -34,6 +34,15 @@ fun ProgressText(
 ) {
   val state = rememberDownloadState(app = app)
 
+  ProgressTextContent(app = app, state = state, showVersionName = showVersionName)
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt:37]: Put arguments on separate lines

File: app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt [hunk 5]
```diff
@@ -57,8 +66,8 @@ fun ProgressText(
     -> if (showVersionName) {
       Text(
         text = text,
-        style = AppTheme.typography.gameTitleTextCondensed,
-        color = AppTheme.colors.secondary,
+        style = AppTheme.typography.body,
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt:69]: In Figma it's `inputs_S` for all cases

File: app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt [hunk 6]
```diff
@@ -84,22 +93,46 @@ fun ProgressText(
...
-      modifier = Modifier
-        .padding(end = 4.dp)
-        .size(16.dp)
+    text = stringResource(string.install_error_short_message),
+    style = AppTheme.typography.inputs_S,
+    color = error
+  )
+}
+
+@PreviewAll
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt:104]: Lets use `PreviewDark`

File: app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt [hunk 7]
```diff
@@ -84,22 +93,46 @@ fun ProgressText(
...
+  AptoideTheme {
+    ProgressTextContent(
+      app = randomApp,
+      state = Installed({}, {}),
+      showVersionName = true
+    )
+  }
+}
+
+@PreviewAll
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt:116]: Lets use `PreviewDark`

File: app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt [hunk 8]
```diff
@@ -84,22 +93,46 @@ fun ProgressText(
...
+      showVersionName = Random.nextBoolean()
     )
-    Text(
-      text = stringResource(string.install_error_short_message),
-      style = AppTheme.typography.bodyCopyXS,
-      color = AppTheme.colors.error
+  }
+}
+
+@PreviewAll
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt:128]: Lets use `PreviewDark`

**Review by G0retZ — CHANGES_REQUESTED** [review]

**Review by G0retZ — CHANGES_REQUESTED** [review]
Installation button doesn't work in SeeMore screen. It immediately reports error instead of download start...

**Review by G0retZ — APPROVED** [review]

**Review by pedroagostinhoaptoide — CHANGES_REQUESTED** [review]
Change the text style of the app items to match the design

**Review by pedroagostinhoaptoide — APPROVED** [review]

**Pull request merged**